src/params/video/Duration.ts

```diff
@@ -0,0 +1,24 @@
+import Action from "../../actions/Action";
+
+/**
+ * @description Defines a video duration.
+ * @namespace Duration
+ * @memberOf Params
+ */
+
+
+/**
+ * Duration
+ */
+class Duration extends Action {
```

Comment:
Is Duration a param or an action?

Reply:
I've removed this class and replaced with a method that ass a new param with key 'du'